README.md

```diff
@@ -87,35 +91,44 @@ docker run \
 -v /etc/logzio-okta/tenants-credentials.yml:/usr/share/logstash/tenants-credentials.yml \
 -t logzio/logzio-okta
 ```
+
 For Mac users: To fix issues with mounting files from root directory please add the path ‘/etc/logzio-okta’ to your Docker File Sharing.
 Click [here](https://medium.com/effy-tech/fixing-the-var-folders-error-in-docker-for-mac-v2-2-3-2a40e776132d) for a guide on how to fix this issue - using docker desktop or manually edit your Docker configuration file.
 For more information about mounting files from root directory click [here](https://docs.docker.com/docker-for-mac/osxfs/#namespaces).
 
 ##### Parameters
 
-| Parameter | Description |
-|---|---|
-| LOGZIO_TOKEN <span class="required-param"></span> | Your Logz.io account [token]((https://app.logz.io/#/dashboard/settings/general)). |
+| Parameter                                                 | Description                                                                                                                               |
+| --------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| LOGZIO_TOKEN <span class="required-param"></span>         | Your Logz.io account [token](<(https://app.logz.io/#/dashboard/settings/general)>).                                                       |
 | LOGZIO_LISTENER_HOST <span class="required-param"></span> | Logz.io [listener URL](https://docs.logz.io/user-guide/accounts/account-region.html) to ship the logs to (for example, listener.logz.io). |
 
-
 ##### Check Logz.io for your logs
 
 Give your logs some time to get from your system to ours,
 and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
 
 ## Versions
 
-0.1.1:
-* Added 'tenant_name' field to the logs
+0.1.2:
```

Comment:
Since we did quite an update to the logstash version, let's bump the version from 0.1.1 to 0.2.0

Reply:
Updated in next commit